### ISSUE TITLE
Second npm install updates package-lock.json with latest semver from package.json

### ISSUE THREAD

**Issue by ghost** [npm/npm#18072]
#### I'm opening this issue because:

  - [ ] npm is crashing.
  - [x] npm is producing an incorrect install.
  - [ ] npm is doing something I don't understand.
  - [ ] Other (_see below for feature requests_):

#### What's going wrong?

First `npm install` sets `required:true` in the `package-lock.json` and does a bunch of other stuff related to that. Second `npm install` does a massive update to the `package-lock.json` instead of reading from it like it's supposed to. It appears to be grabbing the latest semver from `package.json`.

Basically from what I read the package-lock.json is supposed to be read from to avoid these package.json sem ver issues.

### Supporting information:

I'm on 5.2.0 **because I'm getting [this issue](https://github.com/npm/npm/issues/17858) from 5.3.0**. I'm on Ubuntu with Node version 6.11.0. It also happens on 5.1.0.
